anet-dictionary.yml

```
@@ -320,15 +320,9 @@ fields:
       type: enum
       label: UNDEFINED / NATO_UNCLASSIFIED / NATO_UNCLASSIFIED_Releasable_to_EU
       choices:
-        UNDEFINED:
-          value: UNDEFINED
-          label: undefined
-        NATO_UNCLASSIFIED:
-          value: NATO_UNCLASSIFIED
-          label: NATO UNCLASSIFIED
-        NATO_UNCLASSIFIED_Releasable_to_EU:
-          value: NATO_UNCLASSIFIED_Releasable_to_EU
-          label: NATO UNCLASSIFIED Releasable to EU
+        UNDEFINED: undefined
+        NATO_UNCLASSIFIED: NATO UNCLASSIFIED
+        NATO_UNCLASSIFIED_Releasable_to_EU: NATO UNCLASSIFIED Releasable to EU
```

Let's add _Public_ ass discussed, and use abbriviations for classifications.
```suggestion
        undefined: Undefined
        public: Public
        NU: NATO UNCLASSIFIED
        NU_rel_EU: NATO UNCLASSIFIED Releasable to EU
```